test/metabase/lib/js_test.cljs

```diff
@@ -85,3 +86,22 @@
         "should be a JS array")
     (is (= ["collection"]
            (js->clj extras)))))
+
+(deftest ^:parallel template-tags-test
+  (let [db                meta/database
+        metadata-provider (lib.tu/mock-metadata-provider {:snippets {"my snippet" {:snippet-id 19
+                                                                                   :content "where 1 = 1"}}})
+        snippets {"snippet: my snippet"
+                  {:type :snippet
+                   :name "snippet: my snippet",
+                   :id "fd5e96f7-08f8-486b-9919-b2ab72857db4",
+                   :display-name "Snippet: My Snippet",
+                   :snippet-name "my snippet",
+                   :snippet-id 1}}
```

Comment:
Is it important that here we have `:snippet-id 1`  and in the metadata provider `:snippet-id 19`? I mean is this what we expect to encounter in the wild?

Reply:
The mock metadata should have been dropped, snippets are not in metadata at the moment. Perhaps they should be as the FE currently fills in the ids.